src/TodoList/TodoItem/TodoItem.js

```diff
@@ -39,46 +37,46 @@ export class TodoItem extends React.Component {
   closeModifyTodo = () => {
     this.setState({
       editing: false,
-      currentId: 0,
-      currentText: ''
     })
   }
   
  render(){
-    const arrow = this.props.getArrowButton;
-    const status = this.props.todoStatus;
-    const todo = this.props.todo;
-    const {id, text} = todo;
+    const arrowRight = this.props.progressArrow;
+    const arrowLeft = this.props.regressArrow;
 
-   if(this.state.editing) {
-      return(
-        <div className="todo-item">
-          <input
-          type="text"
-          className="todo-input"
-          onChange={this.modifyTodoHandler}
-          value={this.state.currentText}/>
-          <Button
-          clickHandler={this.confirmModifyTodoHandler}
-          classExtra="btn-ok"
-          icon={<FaCheck />}/>
-        </div>
+  if(this.state.editing) {
+    return(
+      <div className="todo-item">
+        <input
+        type="text"
+        className="todo-input"
+        onChange={this.modifyTodoHandler}
+        value={this.state.currentText}/>
+        <Button
+        clickHandler={this.confirmModifyTodoHandler}
+        type="btn-ok"
+        icon={<FaCheck />}/>
+      </div>
       )
     } else {
       return (
         <div className="todo-item">
-          <span className="todo-item-text">{text}</span>
+          <span className="todo-item-text">{this.props.todo.text}</span>
           <Button
-            clickHandler={() => this.openModifyTodo(id, text)}
-            classExtra="btn-edit"
+            clickHandler={this.openModifyTodo}
+            type="btn-edit"
             icon={<FaPencil />}/>
-          <Button
-            clickHandler={() => this.props.changeTodoPlace(id, text, status)}
-            classExtra="btn-arrow"
-            icon={arrow} />
+          {arrowRight ? <Button
+            clickHandler={this.props.progressMove}
+            type="btn-arrow"
+            icon={<FaArrowRight />}/> : null}
+          {arrowLeft ? <Button
+            clickHandler={this.props.regressMove}
```

Comment:
It is better to name event listeners with `on` prefix (like `onClick`) so it is clear that it is a callback. And also try to keep event listeners right before tag closing - it is a good visible separation of attributes and event listeners.